### ISSUE TITLE
Convert-magnet fails

### ISSUE THREAD

**Issue by ilesar** [cpoppema/docker-flexget#72]
I am using showrss to fetch magnets from an rss feed.

When it reaches the magnet conversion step, the task fails due to an incompatible function signature.

```
Unable to convert Magnet URI for entry ********************: Python argument types in


    libtorrent.add_magnet_uri(session, str, add_torrent_params)


did not match C++ signature:


    add_magnet_uri(libtorrent::session {lvalue}, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, boost::python::dict)
```

**Comment by cpoppema** [comment]
Hello @ilesar, thanks for checking in.

This is a known issue with recent versions of the libtorrent library and Flexget; unfortunately the  the underlying operating system for this docker image does not have any older versions available. There is an open issue in the Flexget repository to track this issue.

**Comment by cpoppema** [comment]
For you and everyone else experiencing issues with this: a fix should be available since Flexget version 3.1.94 :)
